src/lib/ndpi_main.c

```diff
@@ -3469,16 +3469,12 @@ u_int16_t ndpi_guess_protocol_id(struct ndpi_detection_module_struct *ndpi_str,
     case NDPI_IPSEC_PROTOCOL_ESP:
     case NDPI_IPSEC_PROTOCOL_AH:
       return(NDPI_PROTOCOL_IPSEC);
-      break;
```

Comment:
Why those `breaks;`s?

Reply:
Sorry,  are you asking why I removed them? They are dead code (just after a `return`) and no compilers (at least, the ones in CI) complains about needing such `break`

Reply:
Sorry, I overlooked that those breaks are actually removed...